seikairooms/urls.py

```diff
@@ -19,5 +19,5 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('rooms.urls')),
-    #path('reservation/', include('reservations.urls')),
+    path('reservation/', include('reservations.urls')),
```

Comment:
this is TINY and NITPICKY, but usually we use plural for these kinds of urls - so `reservations/`. Ignore `admin/` lol, it's its own thing